azure-pipelines/code-coverage/common.yml

```diff
@@ -1,12 +1,17 @@
 # File: azure-pipelines\code-coverage\common.yml
 # Description: Code coverage for common
-# Variable: 'mvnAccessToken' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the variables tab
 # https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
 # Variable: 'commonBranchName' common branch to be used when running the pipeline manually
 name: $(date:yyyyMMdd)$(rev:.r)
 
 trigger: none
 
+pr:
+  branches:
+    include:
+      - '*'
+
```

Comment:
f no pr triggers appear in your YAML file, pull request validations are automatically enabled for all branches, as if you wrote the following pr trigger. This configuration triggers a build when any pull request is created, and when commits come into the source branch of any active pull request.

Reply:
This is the desired behaviour so that's ok.